include/cuco/detail/open_addressing_ref_impl.cuh

```diff
@@ -87,12 +88,14 @@ class open_addressing_ref_impl {
                       ProbingScheme>,
     "ProbingScheme must inherit from cuco::detail::probing_scheme_base");
 
-  static_assert(is_window_extent_v<typename StorageRef::extent_type>,
-                "Extent is not a valid cuco::window_extent");
-  static_assert(ProbingScheme::cg_size == StorageRef::extent_type::cg_size,
-                "Extent has incompatible CG size");
-  static_assert(StorageRef::window_size == StorageRef::extent_type::window_size,
-                "Extent has incompatible window size");
+  /*
+   static_assert(is_window_extent_v<typename StorageRef::extent_type>,
+                 "Extent is not a valid cuco::window_extent");
+   static_assert(ProbingScheme::cg_size == StorageRef::extent_type::cg_size,
+                 "Extent has incompatible CG size");
+   static_assert(StorageRef::window_size == StorageRef::extent_type::window_size,
+                 "Extent has incompatible window size");
+                 */
```

Comment:
@sleeepyjack could use some help here: removing CG size from `window_extent` still cannot unblock those checks. Can you please adjust the example and enable those checks?